lib/template.html

```diff
@@ -97,6 +97,8 @@
   {% assign seo_page_image = seo_page_image | escape %}
 {% endif %}
 
+{% assign seo_page_lang = page.lang | default: site.lang | default: "en_US" %}
```

Comment:
Should there be some sort of string normalization applied with maybe a `| replace: '-', '_'`?

For example it's possible to have a `site.lang` of `en-US` for the `lang` attribute in `<html lang="...">`. Same for `xml:lang` in an Atom feed `<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="...">`.

In both cases it appears they expect values of language-REGION, e.g. `en-US`, where as `og:locale` wants `en_US` with an underscore.

- https://www.w3.org/International/articles/language-tags/
- https://developers.facebook.com/docs/opengraph/guides/internationalization/